### PR TITLE
Feat/login

### PR DESCRIPTION
**Pull request by sedajune**
## Motivation
- implemented login with GitHub
- diy images added
- mongodb works

### CODE REVIEW

File: src/molecules/search-bar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import InputBase from "@mui/material/InputBase";
+import IconButton from "@mui/material/IconButton";
+import SearchIcon from "@mui/icons-material/Search";
+import useStore from "../../ions/hooks/storeFormData";
+
+const SearchBar = () => {
+	const [search, setSearch] = useState("");
+	const results = useStore(state => state.results);
+
+	//useEffect(() => {
```
Comment (Marc-Jamie) [inline src/molecules/search-bar/index.jsx:12]: You could delete this, if you dont need it anymore :)

File: src/organisms/cards/index.jsx [hunk 2]
```diff
@@ -8,11 +8,12 @@ import Typography from "@mui/material/Typography";
 import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
 import ExpandMore from "./styled";
 import FavoriteButton from "../../atoms/checkbox";
-import Map from "../../molecules/map";
+import Image from "next/image";
+import useStore from "../../ions/hooks/storeFormData";
+//import Map from "../../molecules/map";
```
Comment (Marc-Jamie) [inline src/organisms/cards/index.jsx:13]: Same here

File: src/organisms/cards/index.jsx [hunk 3]
```diff
@@ -26,6 +27,8 @@ const PlantCard = ({
 	...rest
 }) => {
 	const [expanded, setExpanded] = React.useState(false);
+	// const [images, setImages] = useState([]);
```
Comment (Marc-Jamie) [inline src/organisms/cards/index.jsx:30]: And here :)

File: src/molecules/search-bar/index.jsx (hunk 1 above)
Comment (sedajune) [reply src/molecules/search-bar/index.jsx:12]: Hahaha but I will definitely still need it ;)

File: src/organisms/image-upload/index.jsx [hunk 4]
```diff
@@ -5,57 +5,72 @@ import PhotoCamera from "@mui/icons-material/PhotoCamera";
...
-						type="file"
-						onChange={upload}
-					/>
-					<IconButton color="primary" aria-label="upload picture" component="span">
-						<PhotoCamera />
-					</IconButton>
-				</label>
-			)}
+			<div>
+				{" "}
```
Comment (Marc-Jamie) [inline src/organisms/image-upload/index.jsx]: Maybe this was from an autocomplete?

File: src/organisms/cards/index.jsx (hunk 2 above)
Comment (sedajune) [reply src/organisms/cards/index.jsx:13]: This might be another feature!

File: src/organisms/image-upload/index.jsx [hunk 5]
```diff
@@ -5,57 +5,72 @@ import PhotoCamera from "@mui/icons-material/PhotoCamera";
...
+				<Image src={localImage}></Image>
+				<input
+					accept="image/*"
+					// style={{ display: "none" }}
```
Comment (Marc-Jamie) [inline src/organisms/image-upload/index.jsx:58]: do you need this?

**Review by Marc-Jamie — APPROVED** [review]
I approve of these changes!

**Pull request merged**